extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -913,7 +919,8 @@ export class DatabaseUI extends DisposableObject {
     ];
     const selectedImportOption =
       await window.showQuickPick<DatabaseImportQuickPickItems>(importOptions, {
-        placeHolder: "Import a database from...",
+        placeHolder:
+          "Import a new database from the cloud or your local machine...",
```

Comment:
I'm not sure that "the cloud" makes sense in this case. Could we perhaps make it something like:

```suggestion
          "Import a new database from GitHub, a URL, or your local machine...",
```

Reply:
I'm happy to update it. I agree just saying "the cloud" is a little unclear.

I copied this text from another location, so I've updated both of them so they stay consistent.